packages/wrangler/e2e/startWorker.test.ts

```diff
@@ -10,7 +10,7 @@ import WebSocket from "ws";
 import { WranglerE2ETestHelper } from "./helpers/e2e-wrangler-test";
 import type { DevToolsEvent } from "../src/api";
 
-const OPTIONS = [{ remote: false }, { remote: true }] as const;
+const OPTIONS = [{ remote: true }, { remote: false }] as const;
```

Comment:
Why this change? I think it was originally the other way round to get a fast fail.